README.md

```diff
@@ -2381,6 +2383,45 @@ Notice that the [`Browser`](#browser) is an immutable object, i.e. this
 method actually returns a *new* [`Browser`](#browser) instance with the
 given setting applied.
 
+#### withHeader()
+
+The `withHeader(string $header, string $value): Browser` method can be used to
+add a request header for all following requests.
+```php
```

Comment:
Can you add an empty line between these two (you already did this in the `withoutHeader()` description)

Reply:
added a new line. :D